tap_hubspot/streams.py

```diff
@@ -522,3 +522,86 @@ class OwnersStream(HubSpotStream):
             ),
         ),
     ).to_dict()
+
+
+### Associations streams
+class DealsAssociationsStream(HubSpotStream):
+  """Deal's associations"""
+
+  name = "deals_associations"
+  path = "/crm/v4/objects/deal/?associations=companies,contacts&limit=100"
```

Comment:
remove `limit`

This is already added here: https://github.com/spacecowboy/tap-hubspot/blob/2ad50cdac9563d293ce1fd997627697dbe53e276/tap_hubspot/client.py#L120